app/src/lib/stores/app-store.ts

```diff
@@ -1503,6 +1509,13 @@ export class AppStore extends TypedBaseStore<IAppState> {
       this.pullRequestStore.getAll(gitHubRepository).then(prs => {
         this.onPullRequestChanged(gitHubRepository, prs)
       })
+
+      // if there's an upstream, we want to add those pull requests, too
```

Comment:
I'm confused, wouldn't the parent repository's PRs already have been added by the previous statement given that `pullRequestStore.getAll` now returns PRs for the parent repository as well?